main.cc

```diff
@@ -1425,6 +1422,9 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 }
             }
 
+            sys_ks.local().build_bootstrap_info().get();
```

Comment:
nit: Can happen as late as the `group0_service.start().get();` happens

Reply:
The idea behind this function (`initialize_system_keyspace`) is the opposite - load things eagerly as early as possible so that we don't have to worry about load order later. Otherwise it's difficult to reason about what should be loaded before what, and comments doesn't help much here.

Reply:
If the function does "load things eagerly as early as possible so that ..." then it shouldn't be named like that

Reply:
What do you suggest?

Reply:
a) Not to introduce `initialize_system_keyspace()` at all
b) (optionally) put call for `sys_ks.local().build_bootstrap_info()` lower in the main() codeflow